deploy/chart/aad-auth-proxy/values-template.yaml

```diff
@@ -12,6 +12,7 @@ aadClientCertificatePath: ""
 aadTokenRefreshIntervalInPercentage: "10"
 otelServiceName: "aad_auth_proxy"
 otelGrpcEndpoint: ""
+overrideRequestHeaders: "{}"
```

Comment:
just curious, is this how a map chart value get configured? Did you consider something like below:
```
overrideRequestHeaders:
- name: "header1"
  value: "value1"
- name: "header2"
  value: "value2;value3"
```

Reply:
This is one way of doing it, I just picked the simpler approach. The one you mentioned might work well with yaml based configuration in helm, not sure if I can do that in yaml for images, where I expect values to be env variables. Also, I thought keeping it generic will work for compiled code running on command line as well.